cumulusci.yml

```diff
@@ -42,25 +42,26 @@ tasks:
             path: ldv_tests/src
         
     deploy_dev_config:
-        description: Deploys the post install configuration for an unmanaged DE org
+        description: Deploys the post-install configuration for an unmanaged DE org
         class_path: cumulusci.tasks.salesforce.Deploy
         options:
             path: unpackaged/config/dev
 
-    deploy_dev_config_managed:
-        description: Deploys the post install configuration for a managed DE org
+    deploy_trial_config:
+        description: Deploys the post-install configuration for a trial/new org.
         class_path: cumulusci.tasks.salesforce.Deploy
         options:
-            path: unpackaged/config/managed
+            path: unpackaged/config/trial
+            namespace_inject: $project_config.project__package__namespace
```

Comment:
Out of curiosity, does this mean it always inserts the namespace where tokens are used? If so, then I'd assume this is only usable in a scratch or other org that has the npsp namespace.

Reply:
@force2b The Deploy task replaces the token with an empty string when the `unmanaged` option is True (the default) or the `namespace_inject` namespace when `unmanaged` is False. So as written here the deploy_trial_config task will set up unmanaged NPSP, but the `config_managed` flow uses this task with `unmanaged` overridden to False to get the namespace for managed NPSP.